docs/shared/reference/nx-json.md

```diff
@@ -126,20 +126,21 @@ like this (which applies to every project):
 }
 ```
 
-And projects can define their prod fileset, without having to redefine the inputs for the `test` target.
+And projects can define their `production` fileset, without having to redefine the inputs for the `test` target.
 
 ```json title="project.json"
 {
   "namedInputs": {
-    "production": [
-      "!{projectRoot}/**/*.test.js",
-      "{workspaceRoot}/jest.config.js"
-    ]
+    "production": ["default", "!{projectRoot}/**/*.test.js"]
   }
 }
 ```
 
-In this case Nx will use the right `prod` input for each project.
+More documentation is available in the [Project Configuration reference](/reference/project-configuration#inputs-&-namedinputs)
```

Comment:
Do you think we can just inline that documentation here? I'm afraid folks might get confused

Reply:
I'd be cautious with that. I know what you mean, but we don't want to bloat the docs again as we did last time. What I'd try to do though is make these type of links more visible. Maybe with a special layout, some sort of button element or something. Because we'll have these type of "link to that other docs page" often

@bcabanes thoughts?

Reply:
I agree that it would be nice to have a visual indicator that we're linking to another part of the docs.  It could be as simple as a 📚icon before each link.

I don't want to have duplicate documentation.  There are a couple ways we could solve this:

1. Leave documentation where it is with inputs and namedInputs being defined in both nx.json and project configuration reference pages.  We'd need clear linking between them.
2. Remove most of the inputs and namedInputs information from those two pages and instead link to a separate inputs/namedInputs reference page that is a reference for the whole system.

Reply:
I came come up with a bit as I did with the "example repository", here is an example:
https://user-images.githubusercontent.com/3447705/193476991-86b9fca6-2dea-4d2e-be8d-c46077b61bff.mov

Reply:
Here is the syntax:
```markdown
{% cards %}
{% card title="Project Configuration reference" description="More documentation is available in a dedicated section." url="/getting-started/intro" /%}
{% /cards %}
```

Here is the result:

https://user-images.githubusercontent.com/3447705/193480021-75c8c7f1-9295-47e6-93a5-712c55437d54.mov

Reply:
Here is a better version with different types:
```markdown
{% cards %}
{% card title="Project Configuration reference" type="documentation" description="More documentation is available in a dedicated section." url="/getting-started/intro" /%}
{% card title="This is an external reference" type="external" description="More documentation is available in a dedicated section." url="https://google.com" /%}
{% /cards %}
```


https://user-images.githubusercontent.com/3447705/193605467-1859fd90-7b24-411c-9834-4b861dc72916.mov